<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="up">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>660</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="sprite.qrc">
      <normaloff>:/img/up.png</normaloff>:/img/up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="down">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>740</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="sprite.qrc">
      <normaloff>:/img/down.png</normaloff>:/img/down.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="left">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>700</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="sprite.qrc">
      <normaloff>:/img/left.png</normaloff>:/img/left.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>700</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="sprite.qrc">
      <normaloff>:/img/right.png</normaloff>:/img/right.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>690</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Restart&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="sprite.qrc">
      <normaloff>:/img/restart.png</normaloff>:/img/restart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QGraphicsView" name="myGraphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>761</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="myStackedWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4">
     <widget class="QStackedWidget" name="menuStack">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>511</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="startButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>160</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>NEW GAME</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>210</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>SAVE PARTI</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>481</width>
          <height>111</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Baba Is You&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Welcome to BABA IS YOU</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </widget>
   </widget>
   <zorder>myGraphicsView</zorder>
   <zorder>up</zorder>
   <zorder>down</zorder>
   <zorder>left</zorder>
   <zorder>right</zorder>
   <zorder>restart</zorder>
   <zorder>myStackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="sprite.qrc"/>
 </resources>
 <connections/>
</ui>
